Sketchbook/CRSF_Joystick/CRSF_Joystick.ino

```diff
@@ -91,7 +91,7 @@ void packetChannels()
       CHANNEL_7_HIGH_EP,         \
       JOYSTICK_LOW,              \
       JOYSTICK_HIGH);
-    gamepad.SetS0(map_data);

```

Comment:
Why are we changing this to SetThrottle instead of leaving it as SetS0

Reply:
Because in the latest version of PicoGamepad there is no S0, instead current Throttle is previous S0, and current S0 is previous S1.
You can check the code here. https://gitlab.com/realrobots/PicoGamepad/-/blob/master/PicoGamepad.cpp#L227

Reply:
Good to know. Thanks for the quick reply!